e2etest/pkg/frr/config/config.go

```diff
@@ -18,30 +18,30 @@ import (
 // BGP router config.
 const bgpConfigTemplate = `
 router bgp {{.ASN}}
+  no bgp ebgp-requires-policy
```

Comment:
curious what issues u faced to do the above I did bgp peering for both iBGP and eBGP w/o the need to this
I thought because we have `no bgp network import-check`, also I assume we are running as datacenter hence its off by default ? `This is enabled by default for the traditional configuration and turned off by default for datacenter configuration.`

Reply:
it seems as of 4.7 the default that network has to exists, I think this need to be debugged further IMHO might be interaction between bgp and rip

Reply:
I think the two things are not related.
`no network import-check` is about being able to advertise networks without having them in the rib.

`no bgp ebgp-requires-policy` is about being able to accept routes when no filters are in place (so it's definetely something needed to add only to the external peer)